docs/getting-started.md

```diff
@@ -30,9 +30,9 @@ First, here are the things you need before starting:
   * USB Ethernet (wired) dongle
   * Bluetooth
 
-  For Raspberry Pi you will need to use the wired Ethernet connection first.
-  With PiStorms the EV3 options will work too since you have a screen to configure
-  them on.
+  For RasberryPi you can use wired Ethernet connection first.
```

Comment:
This sounds a bit weird -- maybe change it to be "For Rasberry Pi, you can use a wired Ethernet connection."